### PR TITLE
Updated XSEDE Notice to Reflect Globus Migration

### PR DESCRIPTION
**Pull request by chakrabortyr**
## Description
Old Text:
screenshot from 2017-11-02 11-59-29

New Text:
screenshot from 2017-11-02 11-59-02

## Motivation and Context
The Globus Migration no longer requires an active XSEDE allocation, so the prior information was no longer accurate.

## Tests performed
Surely.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: html/gui/js/SignUpDialog.js [hunk 1]
```diff
@@ -255,7 +255,7 @@ XDMoD.SignUpDialog = Ext.extend(Ext.Window, {
                     frame: true,
                     title: 'Account Requirements',
                     width: 550,
-                    html: "<p>XSEDE users with an active allocation do not need to request an account and can use their XSEDE credentials. XDMoD accounts are created only for users affiliated with XSEDE, except with the express permission of XSEDE or NSF management.</p>"
+                    html: "<p>Users affiliated with XSEDE do not need to request an XDMoD account; they should use their Globus credentials with a linked XSEDE account*. <br><br>Local accounts are created for users affiliated with XSEDE who do not have an XSEDE User Portal account <b>only with the express permission of XSEDE or NSF management</b>.</p><br><p>* Requires an active XSEDE User Portal account. See <a href=\"https://portal.xsede.org/software/globus\" target=\"_ \">Add XSEDE to your Globus Profile</a> for more information.</p>"
```
Comment (smgallo) [inline html/gui/js/SignUpDialog.js]: Looks like you need to switch to single quotes. You can also get rid of the escaped double quotes then.

File: html/gui/js/SignUpDialog.js [hunk 2]
```diff
@@ -255,7 +255,7 @@ XDMoD.SignUpDialog = Ext.extend(Ext.Window, {
...
                     title: 'Account Requirements',
                     width: 550,
-                    html: "<p>Users affiliated with XSEDE do not need to request an XDMoD account; they should use their Globus credentials with a linked XSEDE account*. <br><br>Local accounts are created for users affiliated with XSEDE who do not have an XSEDE User Portal account <b>only with the express permission of XSEDE or NSF management</b>.</p><br><p>* Requires an active XSEDE User Portal account. See <a href=\"https://portal.xsede.org/software/globus\" target=\"_ \">Add XSEDE to your Globus Profile</a> for more information.</p>"
+                    html: '<p>Users affiliated with XSEDE do not need to request an XDMoD account; they should use their Globus credentials with a linked XSEDE account*. <br><br>Local accounts are created for users affiliated with XSEDE who do not have an XSEDE User Portal account <b>only with the express permission of XSEDE or NSF management</b>.</p><br><p>* Requires an active XSEDE User Portal account. See <a href="https://portal.xsede.org/software/globus" target="_ ">Add XSEDE to your Globus Profile</a> for more information.</p>'
```
Comment (smgallo) [inline html/gui/js/SignUpDialog.js]: The target should literally be `target="_blank"`

**Comment by plessbd** [comment]
It is <a rel="noopener noreferrer" href="https://mathiasbynens.github.io/rel-noopener/"> recommended </a>( <a rel="noopener noreferrer" href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/">performance benefits </a>) to use  `<a href="somesite.com" target="_blank" rel="noopener noreferrer">Link</a>`

**Comment by plessbd** [comment]
why not noreferrer

**Pull request merged**